acceleration-everest-03-dotnet/AppModels/Orders/OrderResultForOtherDtos.cs

```diff
@@ -1,15 +1,15 @@
-﻿using AppModels.Enums;
-using AppModels.Products;
+﻿using AppModels.Products;
+using Infrastructure.CrossCutting.Enums;
 using System;
 using System.ComponentModel.DataAnnotations;
 
 namespace AppModels.Orders;
 
-public class OrderResultOtherDtos
+public class OrderResultForOtherDtos
```

Comment:
```suggestion
public class OrderResult
```
Nomes devem ser autoexplicativos, mas quando autoexplicativos demais pode atrapalhar na interpretação.